ocean_lib/common/ddo/ddo.py

```diff
@@ -20,6 +20,7 @@
 from .constants import DID_DDO_CONTEXT_URL, PROOF_TYPE
 from .public_key_rsa import PUBLIC_KEY_TYPE_RSA, PublicKeyRSA
 from .service import Service
+from ...exceptions import AquariusError
```

Comment:
please do not use relative imports. The reason why relative imports exist here in common, is that we imported common-utils, and we didn't do every SWE thing in the book for time purposes.

Minimally: import the exception using it's full path
For extra effort: adjust older imports to be absolute instead of relative :)